extensions/setup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+import sys
+
+if '..' not in sys.path:
+    sys.path.insert(0, '..')
```

Comment:
Have you tested this much with different installers (e.g. "python setup.py" as well as "pip install <file>")?

sys.path manipulations in a setup.py file give me the heebie-jeebies.